src/api/dataPackagePageApi.js

```diff
@@ -0,0 +1,11 @@
+import axios from "axios";
+
+class DataPackagePageApi {
+
+  static getPackageMetadata(publisher, pack){
+    return axios.get('https://staging.datapackaged.com/api/package/'+publisher + '/' + pack);
```

Comment:
I supposed to go in .env file and from there webpack would pick it up. Will be handled in https://github.com/frictionlessdata/dpr-js/issues/84